src/EditorFeatures/CSharpTest/CodeActions/AddUsing/AddUsingNuGetTests.cs

```diff
@@ -61,10 +61,10 @@ public async Task TestSearchPackageCustomFeedName()
                             .Returns(Task.FromResult(true));
 
         var packageServiceMock = new Mock<ISymbolSearchService>(MockBehavior.Strict);
-        packageServiceMock.Setup(s => s.FindReferenceAssembliesWithTypeAsync("NuGetType", 0, It.IsAny<CancellationToken>()))
-            .Returns(() => ValueTaskFactory.FromResult(ImmutableArray<ReferenceAssemblyWithTypeResult>.Empty));
-        packageServiceMock.Setup(s => s.FindPackagesWithTypeAsync(
-            PackageSourceHelper.NugetOrgSourceName, "NuGetType", 0, It.IsAny<CancellationToken>()))
+        packageServiceMock.Setup(s => s.FindReferenceAssembliesAsync("NuGetType", 0, It.IsAny<bool>(), It.IsAny<CancellationToken>()))
```

Comment:
these members were renamed as they are not 'search for type' specific.  Now they can search for a type/namespace, with a boolean controlling which is being searched for.